extensions/vscode/package.json

```diff
@@ -60,12 +60,14 @@
         "category": "FTL"
       },
       {
-        "command": "ftl.showPanel",
-        "title": "Show Panel"
```

Comment:
Did you mean to remove Show Panel? Just confirming since I wasn't sure :)

Reply:
Yeah the bug was that it needs to be showLogs instead. I should have been more clear on why I changed it!

Reply:
Ahhh sweet thanks a bunch for the context :)